server/mediaserver.go

```diff
@@ -207,7 +207,7 @@ func createRTMPStreamIDHandler(s *LivepeerServer) func(url *url.URL) (strmID str
 				}
 				prof := ffmpeg.VideoProfile{
 					Name:       name,
-					Bitrate:    fmt.Sprintf("%dk", profile.Bitrate),
+					Bitrate:    fmt.Sprint(profile.Bitrate),
```

Comment:
The `fmt.Sprint` is unnecessary here (and a number of other places)

Reply:
`profile.Bitrate` is either an `int32` or `int` in various places, but needs to be a string. Seems like the best way of converting consistently!